salt/metalk8s/addons/ui/config/metalk8s-shell-ui-config.yaml.j2

```diff
@@ -25,12 +34,12 @@ spec:
   canChangeTheme: false
   options:
     main:
-      "https://{{ grains.metalk8s.control_plane_ip }}:8443/":
+      "https://{{ grains.metalk8s.control_plane_ip }}:8443{{ metalk8s_ui_config.spec.basePath }}":
         en: "Platform"
         fr: "Plateforme"
         groups: [metalk8s:admin]
-        activeIfMatches: "https://{{ grains.metalk8s.control_plane_ip }}:8443/(?!alerts).*"
-      "https://{{ grains.metalk8s.control_plane_ip }}:8443/alerts":
+        activeIfMatches: "https://{{ grains.metalk8s.control_plane_ip }}:8443{{ metalk8s_ui_config.spec.basePath }}(?!alerts).*"
+      "https://{{ grains.metalk8s.control_plane_ip }}:8443{{ metalk8s_ui_config.spec.basePath }}/alerts":
```

Comment:
ditto: At least one of those two is wrong (trailing slash)

Reply:
done, I introduced a `normalised_base_path` in the templates which ensures the basePath to be of the shape `/{optional_path}`